.github/workflows/linux.yml

```
@@ -46,9 +52,12 @@ jobs:
       run: |
         export CC=$(which clang)
         export CXX=$(which clang++)
-        python3 -m pip install -U pip
+        python3 -m pip install -U pip pytest
         python3 -m pip install -v .
         python3 -c "import pyamrex; print(pyamrex.__version__)"
+    - name: Unit tests
+      run: |
+        mpiexec -np 1 python3 -m pytest tests/
```

This unit test has no MPI (MPI is off by default for Python atm.):
```suggestion
        python3 -m pytest tests/
```